apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/NormalizedCache.java

```diff
@@ -40,21 +43,24 @@ protected RecordFieldAdapter recordAdapter() {
   }
 
   /**
-   * @param key The key of the record to read
+   * @param key          The key of the record to read
+   * @param cacheHeaders The cache headers associated with the request which generated this record.
    * @return The {@link Record} for key. If not present return null.
    */
-  @Nullable public abstract Record loadRecord(String key);
+  @Nullable public abstract Record loadRecord(String key, Map<String, String> cacheHeaders);
```

Comment:
Should we wrap Map<String, String> into custom object like `CacheHeaders`. So that if we decide to change impl from Map to smth else it will be much easier for refactoring